### DISCUSSION TITLE
Conditional formatting based on string length

### DISCUSSION THREAD

**Discussion by azrikahar** [directus/directus#11531] [Feature Requests]
As a follow up with the conditional formatting implemented in #9773, it'd be nice if we can format the display with the condition that it is a string with NUM length X.

E.g. "If `field_a` is a string with 10 characters then set background color to red"

**Comment by benhaynes** [comment]
Perhaps we could use an INT field when they switch to this? It might not work on database queries, so this would only be for client-side things like the conditional formatting... but that's ok.

**Reply by azrikahar** [reply]
Yea INT field should definitely be the "value" in this case. The slightly trickier part is how to make the dropdown/selection of `string length` appear in the current conditional styling form:

_Note: Here the operators shown are what's available for string. But we'll for sure need to use number operators like gt, gte, lt etc in actuality._

![GB3GPDt08L](https://user-images.githubusercontent.com/42867097/153113218-586b2258-6a67-4ea3-a4e9-9ac5193dfa44.gif)

Perhaps a toggle somewhere around here.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
